wsd/DocumentBroker.cpp

```diff
@@ -4039,17 +4039,23 @@ void DocumentBroker::syncBrowserSettings(const std::string& userId, const std::s
         it.second->updateBrowserSettingsJSON(key, value);
         if (!upload)
```

Comment:
This is reverse logic: if upload is true, we _don't_ upload, but if it's false we _do_ upload.

Better to either rename this to 'uploaded' (past tense) or 'needUpload' (and invert the logic everywhere).

Reply:
I ended taking a bit different route. Is it okay now ?